importer/importer.go

```diff
@@ -0,0 +1,136 @@
+// Package importer is a jsonnet Importer that loads files from https URLs or
+// from the filesystem. It maintains a slice of search paths on which it
+// searches for imports, allowing absolute and relative imports.
+//
+// A search path element may start with a double-slash (`//`) which is
+// interpreted as a HTTPS url without the `https:` scheme prefix. A search
+// path element starting with a single-slash (`/`) is an absolute path
+// in the processes filesystem. A search path without a leading slash
+// is resolved relative to the current working directory of the process.
+//
+// Only HTTPS is supported for network paths.
+
+package importer
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	jsonnet "github.com/google/go-jsonnet"
```

Comment:
I just copied that from another project. I actually wasn't sure what name it would be imported as, so now I know (https://golang.org/ref/spec#Import_declarations).

Fixed in 6edfa3b3b04bf142e32d98ef862ae58c4a0cda37

Reply:
Imo if the package name is not the same as the last directory of the import path it should be a named import, otherwise it might be ambiguous...
ie it is not necessarily obvious that `"github.com/google/go-jsonnet"` imports the `jsonnet` package.

Reply:
Good point. I actually wondered what the import name would be for the directory `go-jsonnet` and from this comment I learnt that the directory name is not relevant to the naming of the package scope. I think then it is useful to specify that name if it is different to the directory name so it is clear what name is being used from looking at the source code.

I will drop that fixup commit.